### PR TITLE
Aswathy/Address details cellxpert v2

### PR DESCRIPTION
**Pull request by aswathy-deriv**
Changes:

-   ...

## Type of change

-   [ ] Bug fix
-   [ ] New feature
-   [ ] Update feature
-   [ ] Refactor code
-   [ ] Translation to code
-   [ ] Translation to crowdin
-   [ ] Script configuration
-   [ ] Improve performance
-   [ ] Style only
-   [ ] Dependency update
-   [ ] Documentation update
-   [ ] Release

### CODE REVIEW

File: src/components/elements/dropdown-search.js [hunk 1]
```diff
@@ -113,6 +118,7 @@ const DropdownSearch = ({
                     />
                     <Arrow onClick={toggleListVisibility} expanded={is_open ? 'true' : 'false'} />
                 </Flex>
+
```
Comment (prince-deriv) [inline src/components/elements/dropdown-search.js:121]: Same here

File: src/pages/signup-affiliates/components/_account-details.tsx [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+    const { send } = useDerivWS()
+
+    useEffect(() => {
+        send(country_list, (response) => {
+            if (!response.error) {
+                type countryType = {
+                    name: string
+                    display_name: string
+                    value: string
+                }
```
Comment (prince-deriv) [inline src/pages/signup-affiliates/components/_account-details.tsx:49]: Let's move this type declaration above

File: src/pages/signup-affiliates/components/_account-details.tsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+                const residence_list_response = response.residence_list.map((el) => {
```
Comment (prince-deriv) [inline src/pages/signup-affiliates/components/_account-details.tsx:50]: let's spread `el` so we can assign it shorter

File: src/components/elements/dropdown-search.js [hunk 4]
```diff
@@ -48,6 +48,7 @@ const DropdownSearch = ({
 }) => {
     const [input_value, setInputValue] = useState('')
     const [dropdown_items, setDropdownItems] = useState([...items])
+
```
[the remark this answers is not in the document]
Comment (aswathy-deriv) [reply src/components/elements/dropdown-search.js:51]: okay

File: src/components/elements/dropdown-search.js (hunk 1 above)
Comment (aswathy-deriv) [reply src/components/elements/dropdown-search.js:121]: okay

**Review by prince-deriv — APPROVED** [review]

**Pull request merged**